src/coreclr/jit/llvmlssa.cpp

```diff
@@ -986,6 +986,10 @@ class ShadowStackAllocator
                     if (m_compiler->lvaInSsa(lclNum))
                     {
                         initValue->SetSsaNum(SsaConfig::FIRST_SSA_NUM);
+                        if (varDsc->lvTracked)
+                        {
+                            VarSetOps::AddElemD(m_compiler, m_compiler->fgFirstBB->bbLiveIn, varDsc->lvVarIndex);
+                        }
                     }
```

Comment:
```suggestion
                    if (m_compiler->lvaInSsa(lclNum))
                    {
                        // Fortunately for the implicit GC def logic, SSA always adds an implicit def for parameters.
                        assert(varDsc->GetPerSsaData(SsaConfig::FIRST_SSA_NUM)->GetDefNode() == nullptr);
                        initValue->SetSsaNum(SsaConfig::FIRST_SSA_NUM);
                    }
                    if (varDsc->lvTracked)
                    {
                        VarSetOps::AddElemD(m_compiler, m_compiler->fgFirstBB->bbLiveIn, varDsc->lvVarIndex);
                    }
```
Not all locals that are tracked have to be in SSA.

The assert is important because, if SSA was more precise, and did not add implicit defs for locals like below:
```
Func(object a) {
   SafePoint();
   a = new object(); // Immediately overwritten. Consider 
}
```
The liveness data would become out of sync with SSA. Liveness would day "this value is used before `a = new object()`", SSA would say "there is no definition of `a` before `a = new object()`". In fact, the existing `SetSsaNum` logic also relies on this.